index.html

```diff
@@ -139,60 +44,152 @@ <h2 class="about-me-h2"> About <br> Myself </h2>
+              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+              <li class="project-style">Back End Dev</li>
+              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+              <li class="project-year">2015</li>
             </ul>
-          </form>
-        </div>
+          <p class="project-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="language-used">
+            <li><div class="tags">html</div></li>
+            <li><div class="tags">css</div></li>
+            <li><div class="tags">javaScript</div></li>
+          </ul>
+        <button type="button" class="button">See Project!</button>
+      </section>
+      <section class="grid-item-1">
+          <img class="work-section-image" src="ressources/Work-3.png" alt="Picture for the first work of Benjamin Patin">
+          <h2>Tonic</h2>
+          <ul class="client-info">
+            <li class="client-name" id="info-client">CANOPY</li>
+            <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+            <li class="project-style"id="info-client">Back End Dev</li>
+            <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+            <li class="project-year"id="info-client">2015</li>
+          </ul>
+          <p class="project-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="language-used">
+            <li><div class="tags">html</div></li>
+            <li><div class="tags">css</div></li>
+            <li><div class="tags">javaScript</div></li>
+          </ul>
+        <button type="button" class="button">See Project!</button>
+      </section>
+      <section class="grid-item-2">
+          <img class="work-section-image" src="ressources/Work-4.png" alt="Picture for the first work of Benjamin Patin">
+          <h2>Multi-Post Stories</h2>
+          <ul class="client-info">
+            <li class="client-name" id="info-client">CANOPY</li>
+            <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+            <li class="project-style"id="info-client">Back End Dev</li>
+            <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+            <li class="project-year"id="info-client">2015</li>
+          </ul>
+          <p class="project-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="language-used">
+            <li><div class="tags">html</div></li>
+            <li><div class="tags">css</div></li>
+            <li><div class="tags">javaScript</div></li>
+          </ul>
+        <button type="button" class="button">See Project!</button>
       </section>
     </section>
-    <footer>
-    </footer>
+
+<!-- About Me Section  -->
+    <section class="about-me">
```

Comment:
About section is not the same as the design. Languages are stacked in one column but they should in a row. `Languages`, `Frameworks`, and `Skills` should have the right arrows. Social icons should have the same design as on the design page.
Yours:
![about](https://user-images.githubusercontent.com/35504149/139495212-2aca9868-7058-4343-afeb-919ea3fb098e.png)

Design:
![about1](https://user-images.githubusercontent.com/35504149/139495305-c72d21f5-36b5-4164-9865-bfde36a17071.png)

Reply:
done :)